Mage.Sets/src/mage/cards/p/PantlazaSunFavored.java

```diff
@@ -0,0 +1,85 @@
+package mage.cards.p;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.EntersBattlefieldThisOrAnotherTriggeredAbility;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.keyword.DiscoverEffect;
+import mage.constants.*;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+
+/**
+ *
+ * @author jimga150
+ */
+public final class PantlazaSunFavored extends CardImpl {
+
+    public PantlazaSunFavored(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{2}{R}{G}{W}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.DINOSAUR);
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(4);
+
+        // Whenever Pantlaza, Sun-Favored or another Dinosaur enters the battlefield under your control,
+        // you may discover X, where X is that creature's toughness. Do this only once each turn.
+        this.addAbility(new PantlazaSunFavoredTriggeredAbility());
+    }
+
+    private PantlazaSunFavored(final PantlazaSunFavored card) {
+        super(card);
+    }
+
+    @Override
+    public PantlazaSunFavored copy() {
+        return new PantlazaSunFavored(this);
+    }
+}
+
+class PantlazaSunFavoredTriggeredAbility extends EntersBattlefieldThisOrAnotherTriggeredAbility {
```

Comment:
Okay, verify failure is because you're missing the copy constructor. But no need to extend this to a custom ability at all. In the card class, just use

```java
this.addAbility(new EntersBattlefieldThisOrAnotherTriggeredAbility(
    new PantlazaSunFavoredEffect(), filter, true, SetTargetPointer.PERMANENT, true
).setDoOnlyOnceEachTurn(true));
```

Also the filter should just be a FilterPermanent, not a FilterCreaturePermanent

Reply:
Ah, ignore my latest commit then, lemme playtest that